network/acp118/handler.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package acp118
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/network/p2p"
+	"github.com/ava-labs/avalanchego/proto/pb/sdk"
+	"github.com/ava-labs/avalanchego/snow/engine/common"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+)
+
+var _ p2p.Handler = (*Handler)(nil)
+
+// Attestor defines whether to a warp message payload should be attested to
+type Attestor interface {
+	Attest(message *warp.UnsignedMessage, justification []byte) (bool, *common.AppError)
```

Comment:
I think we should pass the `context.Context` through into this function (for tracing)